CHANGES.rst

```diff
@@ -1,6 +1,12 @@
 1.5 (unreleased)
 ================
 
+- Add intersect modes for the spatial constraint in the registry module (``pyvo.registry.Spatial``). This is done
+  through a new keyword argument ``intersect`` that can take the following values:
+  * 'covers' keeps the previous behavior that the given region should be covered by the service and is now default,
+  * 'enclosed' is for services covering the given region,
+  * 'overlaps' returns services intersecting with the region. [#495]
+
```

Comment:
I feel this is too verbose for the changelog (keeping only the first sentence should be sufficient). On the other hand not documented in the narrative docs. Would you consider moving things around?

Reply:
Hello, I shortened the changelog and added an example in the docs (where I also included a serticetype constraint example). I was a bit confused about where this should appear in the docs so I hope I got  what you meant?

Reply:
thank you, it all looks good to me.